sycl/ur_win_proxy_loader/CMakeLists.txt

```
@@ -9,7 +9,7 @@ set_property(SOURCE ${CMAKE_CURRENT_BINARY_DIR}/versioninfo.rc
                "RC_INTERNAL_NAME=\"ur_win_proxy_loader\""
                "RC_PRODUCT_NAME=\"ur_win_proxy_loader\""
                "RC_PRODUCT_VERSION=\"${SYCL_VERSION_STRING}\""
-               "RC_COPYRIGHT=\"Copyright (C) 2023 Intel Inc. All Rights Reserved\"")
+	       "RC_COPYRIGHT=\"Copyright (C) 2023-2025 Intel Corporation All Rights Reserved\"")
```

"Latest or Additional Publication Date" is optional, according to our Legal guidelines. Please remove it to avoid updating it each year. 

\+ stylistic fixes after `Intel Corporation`. On the other side, I don't see "All rights reserved" phrase in the copyright notice template in Legal guidelines - I assume it is not needed.


```suggestion
	       "RC_COPYRIGHT=\"Copyright (C) 2023 Intel Corporation.  All rights reserved\"")
```

---

I have dropped the date, and dropped the "all rights reserved", when I thought about it it doesn't even make sense given the license on this repository.